plugin-openshift/che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/OpenshiftServiceClientImpl.java

```diff
@@ -115,6 +115,19 @@ public void makeCall(AsyncCallback<Project> callback) {
     }
 
     @Override
+    public Promise<Void> deleteProject(final String nameSpace) {
+        return newPromise(new AsyncPromiseHelper.RequestCall<Void>() {
+            @Override
+            public void makeCall(AsyncCallback<Void> callback) {
+                String url = openshiftPath + "/project/" + nameSpace;
+                asyncRequestFactory.createDeleteRequest(url)
+                                   .loader(loader, "Deleting project")
```

Comment:
Add '...' in the end of message like https://github.com/codenvy/che-plugins/pull/397/files#diff-159b51602d80c8caa4b8b2a2cc078729R98